docs/user_guide/queue_config.md

```diff
@@ -143,7 +143,13 @@ The name of a queue must be unique at the level that the queue is defined.
 Since the queue structure is fully hierarchical queues at different points in the hierarchy may have the same name.
 As an example: the queue structure `root.testqueue` and `root.parent.testqueue` is a valid structure.
 A queue cannot contain a dot "." character as that character is used to separate the queues in the hierarchy.
-If the name is not unique for the queue in the configuration or contains a dot a parsing error is generated and the configuration is rejected. 
+
+Valid Queue name:
+
+* must be 64 characters or less,
+* could contain alphanumeric character ([a-z0-9A-Z]), underscores (_), colons (:), hashes (#), slashes (/), ampersands (&), and dashes (-).
```

Comment:
@manirajv06 no "ampersands (&)", should it be @?

> var QueueNameRegExp = regexp.MustCompile(`^[a-zA-Z0-9_:#/@-]{1,64}$`)
([Link](https://github.com/apache/yunikorn-core/blob/6939b13d1d0e65aef18722ef546413f3e1775be2/pkg/common/configs/configvalidator.go#L72C1-L73C1))